test/groovy/ArtifactSetVersionTest.groovy

```diff
@@ -39,6 +36,9 @@ class ArtifactSetVersionTest extends BasePipelineTest {
         .around(jsr)
         .around(jer)
 
+    def gitUtils
```

Comment:
[minor] for the sake of small PRs I would like to recommand not to relocate gitUtils and sshAgentList. The smaller the PR, the less background noise, the better and faster to review ...

Reply:
[out of topic] it the ssh agent list used at al?